samples/attested_tls/client/enc/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Generate header with public key of client enclave
+add_custom_command(OUTPUT tls_client_enc_pubkey.h
```

Comment:
This confuses me because it looks like it was done in `attested_tls/CMakeLists.txt`

Reply:
it was a unnecessary block. Removed.